rfc/2022/2022-07-13 - Platform Wide Documentation vs Component Specific Documentation Locations.md

```diff
@@ -0,0 +1,88 @@
+# RFC: Platform-wide VFS developer documentation will be viewed on the VA Platform Website; component-specific technical documentation will be viewed on the Console UI
+
+<!--
+The title is what you want comments on. Use the active voice in a future tense.
+Example:
+    - The website will be built using the XZY framework
+-->
+
+* Comment Deadline: `2022-08-31`
+* Team Crew and Name: [Platform/Spike Teams/Console UI](https://github.com/orgs/department-of-veterans-affairs/teams/platform-pst-console-ui)
+* Authors:
+  * [Thomas Ferrin](https://github.com/TPurple)
+  * [John Chapin](https://github.com/jchapin)
+  * [Susanna Kline](https://github.com/sdk-irl)
+* [Original RFC Pull Request](https://github.com/department-of-veterans-affairs/va.gov-platform-architecture/pull/33)
+
+## Background
+
+The Platform currently has two centralized locations for documentation:
```

Comment:
Since the design system is part of the Platform, should design.va.gov be included here?

Reply:
Yes, it should

Reply:
I have added it in my upcoming review changes.